src/components/search-location/styles.module.scss

```diff
@@ -18,7 +18,12 @@
 
 .searchResultsList {
   z-index: $bring-to-front;
-  @media #{$mobile-only} {
+}
+
+.searchResultsListMobile {
+  z-index: $over-front;
+
+  @media (orientation: landscape) {
```

Comment:
Will this affect other screens if you remove the mobile-only? Maybe we have to have it inside a class too